### ISSUE TITLE
DL3013 rule pip version pinning does not catch `pip3` lines

### ISSUE THREAD

**Issue by ye** [lukasmartinelli/hadolint#48]
This is the current v1.0 version (still supports stdin). As you can see, the `pip3` line does not trigger DL3013 violation check.

```bash
$ hadolint -v
Haskell Dockerfile Linter v1.0

$ hadolint -i <<EOF
> FROM python:3.4
> MAINTAINER Ye Wang <ye@users.noreply.github.com>
> 
> RUN pip3 install requests sqlalchemy
> EOF
```

As long as I changed the `pip3` to `pip`, `hadolint` will pick up that line and reports an violation.
```bash
$ hadolint -i <<EOF
> FROM python:3.4
> MAINTAINER Ye Wang <ye@users.noreply.github.com>
> 
> RUN pip install requests sqlalchemy
> EOF
L4 DL3013 Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>`


```

**Comment by lukasmartinelli** [comment]
Jup only looked for `pip` not `pip3` and `pip2`. Fixed in 3384c5a
